runtime/parachains/Cargo.toml

```diff
@@ -42,6 +43,7 @@ primitives = { package = "polkadot-primitives", path = "../../primitives", defau
 
 rand = { version = "0.8.3", default-features = false }
 rand_chacha = { version = "0.3.1", default-features = false }
+polkadot-runtime-metrics = { path = "../metrics", default-features = false}
```

Comment:
CC

Reply:
can't remove this one, it won't compile.